oeps/processes/oep-0056-architectural-process.rst

```diff
@@ -0,0 +1,218 @@
+
+Missing ceremonies
+------------------
+
+As a community of companies invested in the success of the Open edX platform, we lack any sort of regular ceremonies in which we discuss the platform's architectural priorities as they relate to our desired platform and organization-specific goals.  This often leads to a situation where architectural thinking is done primarily by senior engineering leaders as they learn about their business' needs, and by engineers in the course of projects, with little structured decision making in between to help bridge the highly strategic and the tactical.  Furthermore, the conversations that do happen tend to occur in an ad hoc way - often as "hallway conversations" or one-off project syncs - with whoever can be found to offer enough input to unblock the team.
+
+Specification
+=============
+
+We believe that we can create a healthy and empowering dialogue around our architectural process and mitigate the above concerns by focusing on a re-alignment of ceremonies and clarifying the spirit and process around the creation and review of ADRs.
+
+We will adjust our current architectural process by:
+
+- Formalizing and clarifying an **Architectural Advisory Process** in which we conversationally provide input on and promptly review the status of ADRs, as well as review updates to our architectural principles and tech radar, with the goal of enabling more effective, informed decision making.
+- Improve the discoverability of stakeholders and experts via a **decision maker's checklist**.
+- Documenting as a *suggested best practice* a quarterly **technical strategic review process** where we can review our technical strategy and cross-functional requirements as high-level, guiding input to the Advisory Process, and to help teams and organizations decide where to prioritize addressing "important but not urgent" work such as architectural investments, new capabilities, addressing technical debt, and performing ongoing maintenance/upgrades.
+
+Architectural Advisory Process
+------------------------------
+
+We'll create an Architecture Advisory Process to add structure to today's informal review of ADRs.  The goal of this process is to review ADRs with relevant stakeholders and experts.  The process conversation needs to take place promptly so that decisions do not feel blocked on gathering advice.
+
+We intend to also use the Advisory Process over time to revisit and iterate on our architectural principles with the goal of ensuring they're specific, measurable, achievable, realistic, and testable. (a variation of SMART criteria)  Similarly, like ADRs and architectural principles, we can use the process to review and adjust Open edX Tech Radar blips.
+
+See `Appendix A: Advisory Process Principles <#Appendix A: Advisory Process Principles>`__  for more details on the spirit and functioning of the Advisory Process.
```

Comment:
Appendix A is large. Would it be fair to do a little bit of a summary like

> The Advisory Process is meant to be lightweight and asynchronous. See `Appendix A: Advisory Process Principles <#Appendix A: Advisory Process Principles>`__  for more details on the spirit and functioning of the Advisory Process.

Honestly until getting to the appendix I was thinking it was like, a _process_ which makes me think forms, time, synchronous stuff, etc. I maybe have an outsize aversion to the word. But I do think it is worth clarifying far more upfront than the appendix that this shouldn't be burdensome.

I also don't see anything in here about Slack or Discourse. This feels kind of unapproachable if you're doing it for the first time and want some help.

Reply:
Also if you make an Issue - how do you notify the right people to look at it? I know I don't track issues open in repos I'm following.

Reply:
OK, I see you cover that in `GitHub Groups`. I'm leaving this comment because I may ask for a more crisp summary that hits on more points than your current abstract provides. I really like the TL;DRs in OEP-21 and 25. They are concise bullets that give me a feel for the OEP up front, especially lengthy ones. They anchor my understanding before I dive in. Especially important when there's so much info.

Reply:
I tried to adjust the language a bit to make it seem less imposing.  That said, I mean, the word "process" is the right one here, and the term used in the supporting prior art. :) 

Regarding GH Issues - they're intended as informal super-lightweight first steps.  The DMC would be helpful here too if you want to find some folks to quickly reach out to before trying to write it up into an actual ADR, but I imagine often the initial collaborators on an issue might be teammates who can help frame the initial decision space.

Reply:
(Happy to keep clarifying and wordsmithing this; I think it's important that we make this whole thing as approachable as possible, that's kind of the whole point!)

Reply:
OK, all that makes sense. Since the OEP is fairly long, I don't see a problem with briefly repeating some detail here and there, or pointing at a later section that may provide more details so people know to sit tight with their questions. Here it may be good to link to the section that mentions Issues, GitHub teams, and working with teammates for initial review - I did glean that from later in the OEP, but was reading top to bottom and noting questions that came up as I read.